metis/lib/models/data_block_ledger.rb

```diff
@@ -15,6 +15,14 @@ class DataBlockLedger < Sequel::Model(:data_block_ledger)
     SYSTEM_BACKFILL = 'system_backfill'
     CHECKSUM_COMMAND= 'command_checksum'
 
+    # Environment variable to enable/disable automatic logging
+    # Set METIS_LEDGER_ENABLED=false to disable logging (useful when running backfill first)
+    def self.ledger_enabled
+      env_value = ENV['METIS_LEDGER_ENABLED']
+      return false if env_value.nil?
+      env_value.to_s.downcase == 'true'
+    end
```

Comment:
might be a better practice to put this in config.yml for metis rather than using an ENV variable.